tests/test_buffer_property.py

```diff
@@ -0,0 +1,39 @@
+"Contains a collection of pytest tests that validates the buffer property."
+import random
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import SampleTimingType
+from nidaqmx.errors import DaqError
+from nidaqmx.tests.helpers import generate_random_seed
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestBufferProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the buffer property."""
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_int32_property(self, any_x_series_device, seed):
```

Comment:
I'm sorry, I forgot to bring up NI's regression testing conventions: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions

Using given/when/then and arrange/act/assert would likely require splitting this test case into multiple test cases.